macros/utils/cross_db_utils/current_timestamp.sql

```diff
@@ -2,9 +2,9 @@
   {% if dbt_version >= '1.2.0' %}
```

Comment:
This method moved from dbt utils to dbt core on dbt core 1.3.0
```suggestion
  {% if dbt_version >= '1.3.0' %}
```